recipes/gcc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "4.8.5" %}
+
+package:
+  name: gcc
+  version: {{ version }}
+
+source:
+  fn: gcc-{{ version }}.tar.bz2
+  url: http://www.netgull.com/gcc/releases/gcc-{{ version }}/gcc-{{ version }}.tar.bz2
+  md5: 80d2c2982a3392bb0b89673ff136e223
+
+build:
+  skip: true                             # [win]
+  detect_binary_files_with_prefix: true  # [not linux32]
+  number: 5
+
+requirements:
+  build:
+    # These are taken from the output of the configure scripts
+    - gmp >=4.3.2
+    - mpfr >=2.4.2
+    - mpc >=0.8.1
+    - isl >=0.10,<=0.14
+    - cloog 0.18.0
+    # Do not make gcc a build dependency (you will need to add it to the PATH manually)
```

Comment:
Thanks @jakirkham - I understand now.

Just to not, conda-build doesn't allow bootstrapping anyway, so the comment is somewhat redundant.